python/dataproc_templates/jdbc/README.md

```diff
@@ -249,4 +249,197 @@ export JARS="gs://my-gcp-proj/jars/mysql-connector-java-8.0.29.jar,gs://my-gcp-p
 --jdbctojdbc.output.table="employees_out" \
 --jdbctojdbc.output.mode="overwrite" \
 --jdbctojdbc.output.batch.size="1000"
+```
+
+# 2. JDBC To GCS
+
+Template for reading data from JDBC table and writing into files in Google Cloud Storage. It supports reading partition tabels and supports writing in JSON, CSV, Parquet and Avro formats.
+
+## Arguments
+
+* `jdbctogcs.input.url`: JDBC input URL
+* `jdbctogcs.input.driver`: JDBC input driver name
+* `jdbctogcs.input.table`: JDBC input table name
+* `jdbctogcs.input.partitioncolumn` (Optional): JDBC input table partition column name
+* `jdbctogcs.input.lowerbound` (Optional): JDBC input table partition column lower bound which is used to decide the partition stride
+* `jdbctogcs.input.upperbound` (Optional): JDBC input table partition column upper bound which is used to decide the partition stride
+* `jdbctogcs.numpartitions` (Optional): The maximum number of partitions that can be used for parallelism in table reading and writing. Same value will be used for both input and output jdbc connection. Default set to 10
+* `jdbctogcs.output.format`: Output file format (one of: avro,parquet,csv,json)
```

Comment:
does it support orc format too ? If so please include it.

Reply:
no it doesnt support, i referred. other template too, in all templates they are supporting only these 4 - avro,parquet,csv,json